core/domain/fs_services.py

```diff
@@ -258,15 +274,20 @@ def listdir(self, dir_name):
 
 
 def save_original_and_compressed_versions_of_image(
-        filename, entity_type, entity_id, original_image_content,
-        filename_prefix, image_is_compressible):
+    filename: str,
+    entity_type: str,
+    entity_id: str,
+    original_image_content: bytes,
+    filename_prefix: str,
+    image_is_compressible: bool
+) -> None:
     """Saves the three versions of the image file.
 
     Args:
         filename: str. The name of the image file.
         entity_type: str. The type of the entity.
         entity_id: str. The id of the entity.
-        original_image_content: str. The content of the original image.
+        original_image_content: bytes. The content of the original image.
```

Comment:
In every test case, we are providing byte values ( even in fs_services_tests.py ):
https://github.com/oppia/oppia/blob/e1469c914cd52971aca5970786b502dd2fd3f520/core/domain/fs_services_test.py#L180
https://github.com/oppia/oppia/blob/e1469c914cd52971aca5970786b502dd2fd3f520/core/domain/suggestion_registry_test.py#L2445

Also inside the function, we are using `image_services.compress_image()` method which can only accept bytes. So that's why `original_image_content` is annotated as bytes not str.